### PR TITLE
Update Restraint documentation regarding execution with Beaker

### PR DESCRIPTION
**Pull request by StykMartin**
Small patch to address issues with documentation.
Update Fedora reference a bit + Explain more purpose of custom repo.

Initially, this PR should address #11 but I dropped it in favor of #14. We have to embrace our open source community.

### CODE REVIEW

**Comment by StykMartin** [comment]
F32 / FRawhide blocked by #7

**Comment by StykMartin** [comment]
> Previously it showed fedora-20-x86_64 suffixed in repo name. It may be helpful to have something similar to help people correlate what's needed here. Replace bpeck with some pseudo user name.

This is not really solving the problem. For example if you will use RedHatEnterpriseLinux8 you should use build from epel-8. So, you need to know what you are doing + restraint part of the previous URL is also not correct. For example, I'm not running a restraint project in COPR.

I will revisit docs if I can not add more to it to explain it better instead of trying to hack the URL.

File: docs/using.rst [hunk 1]
```diff
@@ -140,7 +140,13 @@ Sometimes the tests that I am developing can be destructive to the system so I
 don't want to run them on my development box. Or the test is specific to an
 architecture so I can't use a VM for it on my machine. These are cases where
 it's really handy to use a combination of Beaker for provisioning and
-Standalone for executing the tests.
+Standalone for executing the tests. By default, Beaker provides a test harness
+for all imported distributions. You can replace test harness with your build
+by adding a new repository.You can create your build on your own or you can
```
Comment (danrodrig) [inline docs/using.rst:145]: Missing space between "repository.You"

Comment (StykMartin) [reply docs/using.rst]: Wow, you have good :eyes: . Didn't noticed. Fixed.

**Comment by StykMartin** [comment]
> This is kindof unrelated to your changes but do you want to replace /distribution/install with /distribution/check-install?

Sure. We should embrace best practices everywhere.

**Pull request merged**